src/model/case.rs

```diff
@@ -55,3 +55,48 @@ impl Case {
         }
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use crate::model::util::*;
```

Comment:
For ergonomics I usually stick a `use super::*;` at the top of test modules but personal preference